packages/amplify-graphql-rds-model-transformer/rds-lambda/clients/postgres/PostgresClient.ts

```diff
@@ -0,0 +1,67 @@
+import { Knex } from 'knex';
+import { DBClient } from '../DBClient.js';
+import { BaseRequest, Request } from '../../interfaces/BaseRequest.js';
+import { ListRequest } from '../../interfaces/ListRequest';
+
+export abstract class PostgresClient implements DBClient {
+    client: Knex;
+    abstract getClient(): Promise<any>;
```

Comment:
Just use `Promise<Knex>`?

Reply:
good catch I'll update that in the next PR